src/Services/ProcedureService.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
```

Comment:
Is this ProcedureService supposed to be here?

Reply:
Oops It was WIP for Our Next Resource :sweat_smile: